README.md

```diff
@@ -142,9 +154,10 @@ Discussion should be held in the open whenever possible.  However, if you need t
 
 ### Project Resources & Travel Fund
 
-OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md).
+OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md). Details can be found in the [/project-resources](./project-resources) directory.
+
+OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [OLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md).
```

Comment:
Looks like a small typo: the link to the `COLLABORATION_NETWORK.md` doc is missing a "C" at the beginning (currently: `...OLLABORATION_NETW...`)